frontend/src/components/DashboardView/TableBlock.tsx

```diff
@@ -373,6 +376,18 @@ function TableBlock({
                 </Button>
               )}
             </Box>
+            <TextField
```

Comment:
Decided to just expose this field directly, with a default value of 8 (can easily be changed). Playing around with different design options, this felt like the simplest and clearest route to me, but happy to adjust as needed.